.gitignore

```
@@ -1,2 +1,8 @@
 # ignore JetBrains IDE files
 .idea/
+
+# ignore build files
+/pkgbuilds/sunshine/pkg
+/pkgbuilds/sunshine/src
+/pkgbuilds/sunshine/sunshine
+/pkgbuilds/sunshine/*.pkg.*
```

```suggestion
/pkgbuilds/*/pkg
/pkgbuilds/*/src
/pkgbuilds/*/sunshine
/pkgbuilds/*/*.pkg.*
```

To support future packages without having to explicitly add all 4 entries for each one.